From ec322fec7e9c132c4caa0a93175320cb0d8fba73 Mon Sep 17 00:00:00 2001
From: Daniel Mihai <dmihai@microsoft.com>
Date: Mon, 22 Aug 2022 22:02:31 +0000
Subject: [PATCH 3/3] Merged PR 9671: Wait for a possibly slow Guest

Wait for a possibly slow Guest

On some Host VMs it takes longer than 30 seconds to connect to
the Agent - e.g., if enable_debug is enabled for [hypervisor.clh].
---
 src/runtime/config/configuration-clh.toml.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/runtime/config/configuration-clh.toml.in b/src/runtime/config/configuration-clh.toml.in
index f09c095f..0ce7a98d 100644
--- a/src/runtime/config/configuration-clh.toml.in
+++ b/src/runtime/config/configuration-clh.toml.in
@@ -289,7 +289,7 @@ block_device_driver = "virtio-blk"
 
 # Agent connection dialing timeout value in seconds
 # (default: 30)
-#dial_timeout = 30
+dial_timeout = 60
 
 [runtime]
 # If enabled, the runtime will log additional debug messages to the
-- 
2.17.1

